packages/antlr/antlr

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+java -jar $PREFIX/share/antlr/antlr.jar "$@"
```

Comment:
Please don't reference `$PREFIX` variable in Termux packages. Use the absolute path directly. You can use `@TERMUX_PREFIX@` and then sed it to avoid hardcoding.